engine/launcher/src/main/scala/org/enso/launcher/releases/fallback/staticwebsite/FileStorageFallbackReleaseProvider.scala

```diff
@@ -0,0 +1,85 @@
+package org.enso.launcher.releases.fallback.staticwebsite
+
+import org.enso.launcher.releases.fallback.FallbackReleaseProvider
+import org.enso.launcher.releases.{Release, ReleaseProviderException}
+
+import scala.util.control.NonFatal
+import scala.util.{Failure, Success, Try}
+
+/**
+  * Implements a fallback release provider which queries a simple file
+  * [[storage]].
+  *
+  * It has the following requirements:
```

Comment:
You should link to the URL-scheme requirements in the design doc from here.

Reply:
Is it possible to link to a file in Scaladoc? I couldn't find a reliable way to do so, or should I maybe just include a URL to the online docs?